onnxruntime/test/python/transformers/test_flash_attn.py

```diff
@@ -1141,84 +1159,165 @@ def parity_check_gqa_past_no_buff(
     )
 
 
+class TestMHA(unittest.TestCase):
+    def test_packed_mha(self):
+        major, _ = torch.cuda.get_device_capability()
```

Comment:
Need skip test if torch.cuda.is_available() is false. Basically we cannot run it in CPU pipelines.